lockbox-ios/Storyboard/Base.lproj/AccountSetting.storyboard

```diff
@@ -37,11 +37,11 @@
                                     <constraint firstAttribute="height" constant="44" id="mIY-nI-JMy"/>
                                 </constraints>
                                 <fontDescription key="fontDescription" type="system" pointSize="17"/>
-                                <state key="normal" title="Unlink my Account">
+                                <state key="normal" title="Clear out Firefox Lockbox">
```

Comment:
You might have made this change in two places...

Reply:
>You might have made this change in two places...

Sure did. Caught it during merge from master / conflict resolution. 👍